blocks/identity-block/themes/news.json

```diff
@@ -239,22 +239,24 @@
 	"login-form": {
 		"styles": {
 			"default": {
-				"font-family": "var(--font-family-primary)",
+				"font-family": "Inter",
+				"padding-bottom": "var(--global-spacing-5)",
```

Comment:
Like in that other PR, Themes needs to use CSS logical properties in its default styles since we have RTL clients. Please look over https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values and convert any physical property that has a logical equivalent. In this file it seems to be just padding, margin, and width. Note that CSS shorthand props (like `padding`) are not logical, so use `padding-block-start`, `padding-block-end`, etc...